crates/server/src/models/log.rs

```diff
@@ -70,16 +73,22 @@ impl Log {
         Ok(result.rows_affected())
     }
 
-    pub async fn all(pool: &SqlitePool, limit: Option<u32>) -> sqlx::Result<Vec<Log>> {
-        let logs = sqlx::query_as::<_, Log>(r#"SELECT * FROM logs LIMIT ?"#)
-            .bind(limit.unwrap_or(1000))
+    pub async fn list(pool: &SqlitePool, limit: Option<u32>) -> sqlx::Result<Vec<Log>> {
```

Comment:
This log structure and associated methods, particularly 'list' order logs by 'id DESC' which is a sensible default for most log viewing purposes.